lib/server/web.js

```diff
@@ -39,8 +39,18 @@ function trimLocale(header) {
 // This is the webserver. It's what the outside always talks to. It
 // handles the whole Profile API.
 exports.create = function createServer() {
+  var useRedis = config.serverCache.useRedis;
+  var cache = {
+    engine: useRedis ? require('catbox-redis') : require('catbox-memory')
+  };
+  if (useRedis) {
+    cache.host = config.serverCache.redis.host;
+    cache.port = config.serverCache.redis.port;
+    cache.partition = config.serverCache.redis.partition;
+  }
   var isProd = config.env === 'production';
   var server = new Hapi.Server({
+    cache: cache,
```

Comment:
@udaraweerasinghege could we please add a config for `useCache` means that if `useCache` is `false` then the `cache` setting here would be `null` or `undefined`.  (Rephrased, if `false` then **none** should be used `require('catbox-redis') : require('catbox-memory')`.

 You can set `useCache` to `true` in the main config.js file.